Without this patch, a compiler error happens if one tries to add
more 'case' clauses after TopAbs_VERTEX.

From 5d73a6188afdb7d084a44a617c3f65e0642fe321 Mon Sep 17 00:00:00 2001
From: QbProg <tholag@gmail.com>
Date: Sun, 24 Jul 2011 11:20:39 +0200
Subject: [PATCH] Fix buid failure introduced by 82de4a4

---
 src/DNaming/DNaming_TransformationDriver.cxx |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

Index: oce/src/DNaming/DNaming_TransformationDriver.cxx
===================================================================
--- oce.orig/src/DNaming/DNaming_TransformationDriver.cxx
+++ oce/src/DNaming/DNaming_TransformationDriver.cxx
@@ -320,6 +320,7 @@ static void CollectShapes(const TopoDS_S
     }
     break;
   case TopAbs_VERTEX:
+  {
     const Handle(TNaming_NamedShape) aNS =  TNaming_Tool::NamedShape(SSh, theLab);
     if(!aNS.IsNull())
       if(SMap.Add(SSh)) {
@@ -327,6 +328,7 @@ static void CollectShapes(const TopoDS_S
 //	if(isPrimitive)
 //	  TagMap.Bind(SSh, aNS->Label().Tag());
       }
+  }
     break;
   }   
 }
